docs/DeveloperWorkflow/automated_checks.md

```diff
@@ -0,0 +1,38 @@
+_DeveloperWorkflow:
```

Comment:
The formatting needs correction. Check the preview.

Reply:
Done